src/pages/home/sidebar/ProfileAvatarWithIndicator.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import {View} from 'react-native';
+import type {OnyxEntry} from 'react-native-onyx';
+import {withOnyx} from 'react-native-onyx';
+import AvatarWithIndicator from '@components/AvatarWithIndicator';
+import OfflineWithFeedback from '@components/OfflineWithFeedback';
+import useCurrentUserPersonalDetails from '@hooks/useCurrentUserPersonalDetails';
+import useThemeStyles from '@hooks/useThemeStyles';
+import * as UserUtils from '@libs/UserUtils';
+import ONYXKEYS from '@src/ONYXKEYS';
+
+type ProfileAvatarWithIndicatorOnyxProps = {
+    /** Indicates whether the app is loading initial data */
+    isLoading: OnyxEntry<boolean>;
+};
+
+type ProfileAvatarWithIndicatorProps = ProfileAvatarWithIndicatorOnyxProps & {
+    /** Whether the avatar is selected */
+    isSelected: boolean;
+};
+
+function ProfileAvatarWithIndicator({isLoading = true, isSelected = false}: ProfileAvatarWithIndicatorProps) {
+    const styles = useThemeStyles();
+    const currentUserPersonalDetails = useCurrentUserPersonalDetails();
+
+    return (
+        <OfflineWithFeedback pendingAction={currentUserPersonalDetails.pendingFields?.avatar ?? null}>
```

Comment:
```suggestion
        <OfflineWithFeedback pendingAction={currentUserPersonalDetails.pendingFields?.avatar ?? null}>
```

Do we need this `?? null`?